server/build/webpack.js

```diff
@@ -45,7 +45,7 @@ export default async function createCompiler (dir, { hotReload = false, dev = fa
     new webpack.optimize.CommonsChunkPlugin({
       name: 'commons',
       filename: 'commons.js',
-      minChunks: pages.length
+      minChunks: pages.length + 1
```

Comment:
This is not for that. We put this as the number of pages because, if something is common to all pages, it will move to the commons.js